department-of-veterans-affairs/va.gov-cms
# VACMS-7022 refactor old theme code

## Description

Relates to #7022. 

- All ported theme code has been updated to use the new color vars whenever possible. 
- Icon paths fixed for new theme structure. 
- Main styles.scss file organized out into smaller parts for future ease of maintainability. It looks like a lot was removed/added, but mostly it was shuffled around.
- Updated READMEs, added comments to the top-level standalone .scss files (styles/proofing/whats_new/etc)
- Added template override for node link report to apply consistent alert styles. This could be removed with some updates to the contrib module, but for now this should do :)

Some of these ported code bits we may ultimately decide to revert in design (e.g. uppercase labels on node view pages?) but for now I kept things as close to the spirit of what it was as possible.

## Testing done
Looked at lots and lots of node edit forms, node view pages, content lists, centralized content, kb pages as an admin and an editor.

## Screenshots
<img width="963" alt="Screen Shot 2021-12-20 at 3 21 43 PM" src="https://user-images.githubusercontent.com/11279744/146845983-afda5c7d-57af-4fa1-bfff-d73cd519f7b6.png">
<img width="606" alt="Screen Shot 2021-12-20 at 3 23 24 PM" src="https://user-images.githubusercontent.com/11279744/146845986-0c053bdf-af33-4ac9-a2a7-65ae93cb22c7.png">
<img width="1916" alt="Screen Shot 2021-12-20 at 3 25 03 PM" src="https://user-images.githubusercontent.com/11279744/146845987-e1e69f86-8cd0-4aac-9dbe-85260f1dd352.png">


## QA steps
log in as an admin, enable vagovclaro as admin theme, click around and verify things don't look too out of whack.

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
